roles/openshift_node/vars/main.yml

```diff
@@ -2,3 +2,4 @@
 openshift_node_config_dir: "{{ openshift.common.config_base }}/node"
 openshift_node_config_file: "{{ openshift_node_config_dir }}/node-config.yaml"
 openshift_version: "{{ openshift_pkg_version | default('') }}"
+openshift_node_proxy_mode: "{{ openshift_node_proxy_mode | default('iptables') }}"
```

Comment:
Ansible doesn't allow for redefining a variable like this.

Instead we should persist this value using openshift facts in roles/openshift_node/tasks/main.yml, then the template could reference openshift.node.proxy_mode.